tests/lib/prepare-project.sh

```diff
@@ -121,14 +121,21 @@ if [[ "$SPREAD_SYSTEM" == ubuntu-14.04-* ]]; then
     quiet apt-get install -y --force-yes apparmor libapparmor1 seccomp libseccomp2 systemd cgroup-lite util-linux
 fi
 
-distro_purge_package snapd
-# utilities
-# XXX: build-essential seems to be required. Otherwise package build
-# fails with unmet dependency on "build-essential:native"
-distro_install_package build-essential curl devscripts expect gdebi-core jq rng-tools git netcat-openbsd
-
-# in 16.04: apt build-dep -y ./
-quiet apt-get install -y $(gdebi --quiet --apt-line ./debian/control)
+. "$TESTSLIB/distro.sh"
```

Comment:
This looks like a nice abstraction. However I wonder if it would be better to move this into pkgdb.sh as this is currently the central place of knowledge about pkg names on various distros.